.github/workflows/reusable-oci-image-push.yml

```
@@ -0,0 +1,133 @@
+name: Build kustomize oci artifact and push to altinncr
+description: |
```

actionlint (1.7.4)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)